### ISSUE TITLE
Mapper: display CSV file structure

### ISSUE THREAD

**Issue by martinmaillard** [zazuko/cube-creator#14]
The mapper needs to show the structure of a CSV file.

It shows:
- The name of the CSV file
- The name of each column
- For each column, a small sample of the values it contains

<img width="701" alt="Screenshot 2020-09-15 at 16 16 29" src="https://user-images.githubusercontent.com/1334124/93222234-037a5200-f76f-11ea-886c-1b829c0db9dc.png">

Depends on #62

**Comment by tpluscode** [comment]
What are the colourful dots?

**Comment by martinmaillard** [comment]
As shown [in the mockups](https://zazuko.github.io/bafu-datacube-workshop2-mockups/#/cubes/quality-of-air-ch/mapper), it shows which columns are already mapped. But this is not part of this task.
